docs/sdks/c-sharp/index.md

```diff
@@ -498,7 +498,7 @@ class EventContext {
 }
 ```
 
-The `Reducers` property of the context provides access to reducers exposed by the remote module. See [Observe and invoke reducers](#observe-and-invoke-reducers).
+The `Reducers` property of the context provides access to reducers exposed by the remote database. See [Observe and invoke reducers](#observe-and-invoke-reducers).
```

Comment:
```suggestion
The `Reducers` property of the context provides access to reducers exposed by the remote module. See [Observe and invoke reducers](#observe-and-invoke-reducers).
```

Reducers are a property more of the module than the database I think it's fine to just say module here.